app/containers/Header/index.js

```diff
@@ -0,0 +1,133 @@
+/**
```

Comment:
Navigation should be a component without logic. I do not think it is a container. Sign in will be function of separate screen / container.